### PR TITLE
Improve Ergo Jr page.

### PR DESCRIPTION
**Pull request by pierre-rouanet**
I've added some information on the Ergo Jr page to work on #22 . Let me know what you think.

@damiencaselli I did not succeed to integrate the assembly gif.

### CODE REVIEW

**Comment by pierre-rouanet** [comment]
I also don't really get why travis is unhappy: https://travis-ci.org/poppy-project/poppy-project.org/builds/133949837#L240

> -  error parsing attribute name (line 1)

File: _pages/robots/ergo/en.html [hunk 1]
```diff
@@ -24,3 +24,42 @@
...
+<p>Ergo Jr relies on <a href="http://snap.berkeley.edu">Snap!</a> a variant of <a href="https://scratch.mit.edu">scratch</a> the visual programming language. Its accessibility and efficiency make it a great tool to discover robotics and understand the digital world. You can use it directly on the robot and do not need to install anything.</p>
+
+<img src="{{ site.baseurl }}/assets/img/snap/simple-example.jpg" alt="Simple If Then example in Snap" width="1080" height="637">
+
+<p>It comes with a set of pedagogical activies, developed with high-school teachers, ranging from the discovery of the robot and how to control it to more complex and integrated project. They can be accessed <a href="https://forum.poppy-project.org/tags/c/education/pedagogical-activities/ergo-jr">here</a>. Some examples can be seen in the videos below:</p>
+
+<div class="video-showcase">
+  <div class="row">
+    <div class="large-6 columns">
+      <div class="flex-video"->
```
Comment (ghost) [inline _pages/robots/ergo/en.html]: `<div class="flex-video"->` should be `<div class="flex-video">`

**Comment by ghost** [comment]
@pierre-rouanet I've reviewed text, good job 👍 
I'll add this to the master and translate it to French.

**Comment by pierre-rouanet** [comment]
Thanks for the review @damiencaselli. I'm agree with all of your improvements!

**Pull request merged**